NewForce-Cohort-9/nutshell-react-jurassic-attack
# Event and Task -cb branch1

# Added
- Events, Task  components to hold all the jsx modules for the TaskEventContainer
- Forms component to hold the Event and Task forms for both edit and create options
- databse.json: Task Array now has a "completedByDate:" object

# Testing Instructions

1. After logging in, click on the Task and Events on the NavBar. 
2. See all the current Events and Tasks 
3. Click the Create New Event button
4. See the form the create new Event
5. Fill the form out and submit
6. Check the Task and Event page for new Event
7. Check database.json that you new Event has been saved
